accept.go

```diff
@@ -62,6 +63,18 @@ type AcceptOptions struct {
 	// Defaults to 512 bytes for CompressionNoContextTakeover and 128 bytes
 	// for CompressionContextTakeover.
 	CompressionThreshold int
+
+	// OnPingReceived is an optional callback invoked synchronously when a ping frame is received.
+	//
+	// To avoid blocking, process the callback asynchronously using a goroutine.
+	// If you need to reuse the payload outside the callback, clone the byte slice.
+	// Any modifications to the payload within the callback will be sent in the subsequent pong frame.
+	OnPingReceived func(context.Context, []byte)
+
+	// OnPongReceived is an optional callback invoked synchronously when a pong frame is received.
+	//
+	// To avoid blocking, process the callback asynchronously using a goroutine.
```

Comment:
```suggestion
	// To avoid blocking, process the callback asynchronously using a goroutine.
	// If you need to reuse the payload outside the callback, clone the byte slice.
	// Modifying the payload is not supported and will break Ping.
```
There's an asymmetry between these two callbacks that I think would be good to document, wdyt?

Reply:
There's even more asymmetry now, I tried to improve that part of the docs